site/viamguides/src/ncnn-edge-ai/ncnn-edge-ai.md

```diff
@@ -0,0 +1,246 @@
+
+1. At the bottom of the `camera-1` panel, expand the **TEST** section to ensure you have configured the camera properly and see a video feed.
+   ![Test camera feed](./assets/camera-test.png)
+
+<!-- ------------------------ -->
+## Add and configure the ncnn ML model
+Duration: 10
+
+Now, let's add the ncnn ML model service to our machine. This will let us run optimized neural networks for computer vision.
+
+### Add the ML model service
+
+1. In [the Viam app](https://app.viam.com/fleet/locations) under the **CONFIGURE** tab, click the **+** icon in the left-hand menu and select **Service**.
+1. Select `ML Model`, and find the `hipsterbrown:mlmodel:ncnn` module. This adds the module for running optimized neural networks with ncnn. Leave the default name `mlmodel-1` for now.
+   ![find ncnn ML model service](./assets/find-mlmodel.png)
+   ![add ncnn ML model service](./assets/add-mlmodel.png)
+1. Notice adding this service adds the ML model service called `mlmodel-1`.
+   ![new ML model service](./assets/new-mlmodel-service.png)
+1. In the new `mlmodel-1` panel, configure your service with the following JSON in the **Attributes** section:
+   ```json
+   {
+       "model_name": "squeezenet_ssd",
+       "num_threads": 4
+   }
+   ```
+   This tells the ML model service to use the pre-trained "squeezenet_ssd" model from the ncnn model zoo and allocates 4 CPU threads for inference.
+   
+   > aside positive
+   > The "squeezenet_ssd" model is a lightweight object detection model that can identify 21 different object categories.
+   >  "background"
+   >  "aeroplane"
+   >  "bicycle"
+   >  "bird"
+   >  "boat"
+   >  "bottle"
+   >  "bus"
+   >  "car"
+   >  "cat"
+   >  "chair"
+   >  "cow"
+   >  "diningtable"
+   >  "dog"
+   >  "horse"
+   >  "motorbike"
+   >  "person"
+   >  "pottedplant"
+   >  "sheep"
+   >  "sofa"
+   >  "train"
+   >  "tvmonitor"
+   ![configure mlmodel service](./assets/mlmodel-service-config.png)
+
+1. Click **Save** in the top right to save and apply your configuration changes.
+
+<!-- ------------------------ -->
+## Configure the Vision Service
+Duration: 5
+
+Now, let's add a vision service that will use our ML model to perform object detection on images from the webcam.
+
+1. In [the Viam app](https://app.viam.com/fleet/locations) under the **CONFIGURE** tab, click the **+** icon in the left-hand menu and select **Service**.
+1. Select `vision`, and find the `ML Model` module. This adds a vision service that uses ML models. Leave the default name `vision-1` for now.
+   ![Add vision service](./assets/find-vision.png)
+1. Notice adding this service adds the vision service called `vision-1`.
+   ![new vision service](./assets/new-vision-service.png)
+1. In the **ML Model** section of the panel, select your ML model service (`mlmodel-1`).
+1. In the **Depends on** section, select your camera component (`camera-1`).
```

Comment:
In the previous screenshot, I see you updated your confidence to 0.5. If it helps people have a better hello-world experience, I recommend explicitly adding that as a step.